docs/future-work/cmp-hierarchy-improvements/add-top-down-connections.md

```diff
@@ -6,4 +6,6 @@ In Monty systems, low-level LMs project to high-level LMs, where this projection
 
 For example, a high-level LM of a dinner-set might have learned that the fork is present at a particular location in its internal reference frame. When at that location, it would therefore predict that the low-level LM should be sensing a fork, enabling the perception of a fork in the low-level LM even when there is a degree of noise or other source of uncertainty in the low-level LM's representation.
 
-In the brain, these top-down projections correspond to L6 to L1 connections, where the synapses at L1 would support predictions about object ID. However, these projections also form local synapses en-route through the L6 layer of the lower-level cortical column. In a Monty LM, this would correspond to the top-down connection predicting not just the object that the low-level LM should be sensing, but also the specific location that it should be sensing it at. This could be complemented with predicting a particular pose of the low-level object (see [Use Better Priors for Hypothesis Initialization](../learning-module-improvements/use-better-priors-for-hypothesis-initialization.md)).
\ No newline at end of file
+In the brain, these top-down projections correspond to L6 to L1 connections, where the synapses at L1 would support predictions about object ID. However, these projections also form local synapses en-route through the L6 layer of the lower-level cortical column. In a Monty LM, this would correspond to the top-down connection predicting not just the object that the low-level LM should be sensing, but also the specific location that it should be sensing it at. This could be complemented with predicting a particular pose of the low-level object (see [Use Better Priors for Hypothesis Initialization](../learning-module-improvements/use-better-priors-for-hypothesis-initialization.md)).
+
+This location-specific association in both models is key to how we believe compositional objects are represented. For example, if you had a coffee mug with a logo on it, that logo might make an (unusual) 90 degree bend half-way along its length. This could be learned by associating the logo with the mug twice, where different locations in the logo space, as well as different poses of the logo, would be associated with the logo depending on the location on the mugs surface.
```

Comment:
As far as I understood it so far, the logo is always associated with the mug multiple times at multiple locations, even if it does not have the bend. Also when it has the bend, there are likely more than two associations.

Reply:
Yeah fair point, I was phrasing it in the more toy situation, but it's probably better to write it as you suggest.